source/part1/chapter-02/md/00-pandas-basics.md

```diff
@@ -36,41 +36,53 @@ Pandas supports reading and writing data e.g. from/to:
 - Python Pickle format
 - SQL (Postgresql, MySQL, Oracle, MariaDB, etc.)
 
-See full list from the latest Pandas documentation [^urlpandasdocs].
+See full list from the latest pandas documentation [^urlpandasdocs].
 
 <!-- #region deletable=true editable=true -->
 ## Pandas data structures
 
-In Pandas, table-like data are stored in two-dimensional DataFrames with labeled rows and columns. You can think of the pandas DataFrame as a programmable spreadsheet. The Pandas DataFrame was originally inspired by dataframes that are in-built in the R programming language. One-dimensional sequences of values are stored in pandas Series. One row or one column in a Pandas DataFrame is actually a Pandas Series. You can think of a pandas Series as a clever list. These Pandas structures incorporate a number of things we've already encountered, such as indices, data stored in a collection, and data types.
+In pandas, table-like data are stored in two-dimensional DataFrames with labeled rows and columns. You can think of the pandas DataFrame as a programmable spreadsheet. The pandas DataFrame was originally inspired by dataframes that are in-built in the R programming language. One-dimensional sequences of values are stored in pandas Series. One row or one column in a pandas DataFrame is actually a pandas Series. You can think of a pandas Series as a clever list. These pandas structures incorporate a number of things we've already encountered, such as indices, data stored in a collection, and data types.
 
 ![Pandas data structures](./../img/pandas-structures.png)
 
 
 ![Pandas data structures annotated](./../img/pandas-structures-annotated.png)
 
-*Pandas DataFrame is a 2-dimensional data structure used for storing and mainpulating table-like data (data with rows and columns). Pandas Series is a 1-dimensional data structure used for storing and manipulating an sequence of values.*
+*pandas DataFrame is a 2-dimensional data structure used for storing and mainpulating table-like data (data with rows and columns). pandas Series is a 1-dimensional data structure used for storing and manipulating an sequence of values.*
```

Comment:
Capitalize "Pandas" at the start of sentence?

Reply:
Hmm, here in the bullet lists, I would perhaps keep lower case even when it starts..